index.html

```diff
@@ -0,0 +1,87 @@
+<!DOCTYPE html>
+<html lang="ko">
+	<head>
+		<meta charset="UTF-8">
+		<meta name="viewport" content="width=device-width, initial-scale=1.0">
+		<link rel="stylesheet" href="style.css">
+		<title>판다마켓</title>
+	</head>
+	<body>
+		<!-- header section / 로고, 로그인 -->
+		<header>
+			<div class="banner">
+				<div class="banner-main">
+					<a class="header-logo" href="index.html">
+						<img src="./images/header-logo.png" alt="로고">

```

Comment:
💊 제안

img 태그의 alt 속성은 alternative 라는 의미로, 이미지 파일을 다운로드하는 것에 실패해서 이미지 파일을 보여줄 수 없을 때
어떤 이미지인지 파악할 수 있게 대신 제공되거나, 스크린리더로 읽혀지는 문자를 의미합니다.
따라서 가능한 해당 이미지를 설명하는 내용을 넣어주면 좋습니다~
지금의 경우 해당 이미지가 안 보일때 제공되면 좋을 텍스트는 이미지에 들어간 문자인 "판다마켓"이 되겠네요~

```suggestion
						<img src="./images/header-logo.png" alt="판다마켓">

```